variables.tf

```diff
@@ -61,3 +61,46 @@ variable "ssh_user" {
 variable "welcome_message" {
   default = ""
 }
+
+variable "comparison_operator" {
+  description = "The arithmetic operation to use when comparing the specified Statistic and Threshold. Possible values are: GreaterThanOrEqualToThreshold, GreaterThanThreshold, LessThanThreshold, LessThanOrEqualToThreshold."
```

Comment:
Use these descriptions in README.md.

Reply:
if add it to readme the line will be to long for table and markdown'll become unreadable 
<img width="989" alt="screenshot 2017-09-01 18 54 22" src="https://user-images.githubusercontent.com/26582191/29977726-0eb8d7c4-8f47-11e7-8942-49869636bc1e.png">